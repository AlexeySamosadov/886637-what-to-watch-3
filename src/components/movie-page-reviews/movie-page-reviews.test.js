import React from "react";
import renderer from "react-test-renderer";
import MoviePageReviews from "./movie-page-reviews.jsx";

const filmData = {
  name: `The Grand Budapest Hotel`,
  date: 2014,
  genre: `Drama`,
  src: `somePath`,
  id: `55`,
  srcPoster: `somePath`,
  ratingCount: 323,
  ratingLevel: `mustSee`,
  description: `dsfsdfdsfsfsdfsdfdsf`,
  actors: [`Toni Hawk`, `Arnold Vicci`, `Van gog`, `Charli Chaplin`, `German French`, `Italian English`],
  directors: `Alex Smitch`,
  srcVideo: `somePath`,
  comments: [{
    commentId: `id734832`,
    commentText: `Some text`,
    commentatorName: `Antoni Banderers`,
    commentRating: 7.9,
    commentTime: new Date(1466424490000),
  }]
};

it(`This is Unit Test for MoviePageDetails component`, ()=>{
  const tree = renderer
    .create(
        <MoviePageReviews
          filmData={filmData}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
