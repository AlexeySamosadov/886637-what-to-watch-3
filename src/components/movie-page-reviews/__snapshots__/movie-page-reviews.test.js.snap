// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is Unit Test for MoviePageDetails component 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Antoni Banderers
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-20"
          >
            5
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        7.9
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
